pkg/kyma/docs/install.md

```diff
@@ -14,17 +14,17 @@ kyma install [OPTIONS]
 
 | Name     | Short Name | Default value| Description|
 | ----------|---------|-----|------|
-| --release | -r ||Kyma release to use|
-| --local | -r |false|Install from sources| 
-| --config | -r ||URL or path to the Installer configuration YAML file| 
-| --override | -r ||Path to YAML file with parameters to override. Multiple entries of this flag are allowed| 
-| --domain | -r |kyma.local|Domain used for installation| 
-| --password | -r ||Predefined cluster password| 
-| --noWait | -r |false|Do not wait for the Installer configuration to complete| 
-| --src-path | ||Path to local sources| 
-| --installer-version | ||Version of the Kyma Installer Docker image used for local installation| 
-| --installer-dir | ||The directory of the Kyma Installer Docker image used for local installation| 
-| --timeout |  |0|Timeout after which CLI stops watching the installation progress| 
+| --release | -r ||Specifies the Kyma release or git revision to be installed. Go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about each of the available releases, or use the revision of your choice. For example, `kyma install --release master`.|
+| --local | -r |false|Indicates installtion from sources. To make it work, make sure you have **{GO_PATH}** set and the location of your sources complies with the Go file and naming convention.| 
```

Comment:
installation from a local Kyma git checkout. If the checkout is located following the XXX conventions, the CLI will find it automatically, otherwise use --XX to configure the path explicit

Reply:
Done.